tests/main/snap-run/task.yaml

```
@@ -1,5 +1,10 @@
 summary: Check that `snap run` runs
 
+details: |
+    Verifies the snap run command and its parameters. Check
```

just for consistency with the others

```suggestion
    Verifies the `snap run` command and its parameters. Check
```